### DISCUSSION TITLE
How to add form inside ViewAction infolist ?

### DISCUSSION THREAD

**Discussion by aryadilas** [filamentphp/filament#10710] [Help]
I have ParticipantResource and I want to add action in the table to see the participant detail and to register participant, so i tried
```
Tables\Actions\ViewAction::make()
    ->label('Detail')
    ->modalWidth('5xl')
    ->infolist([

         //Another section to show participant detail

        Section::make('Registration')
            ->description('Participant Registration')
            ->icon('heroicon-c-identification')
            ->collapsible()
            ->hidden(fn () => !auth()->user()->hasRole('adminlv1') ? false : true )
            ->schema([
            ])
            ->columns(1),
    ]),
```
   

I succeded to show participant detail with TextEntry , but how to add form schema inside the infolist section?
I have been searching in documentation but still cant find what i need..

**Comment by ShaungBhone** [comment]
Do you mean https://filamentphp.com/docs/3.x/infolists/actions#adding-an-action-to-a-custom-infolist-component?

**Reply by aryadilas** [reply]
I dont know, I'm confused when implementing that feature, because ViewEntry need view() in it,, and I don't know how to prepare that,

**Reply by ShaungBhone** [reply]
Make a button and create a modal action or sth like that.

**Reply by aryadilas** [reply]
what variable should exists inside the view?
I try $slot , but causing an error.
have you try that? can you give example?
